docs/themes.md

```diff
@@ -28,13 +28,32 @@ OPTIONS
   --port=port      [default: 4567] Port for the http server to use
   --logs           Tail logs
   --no-livereload  Disable live-reloading the preview when a change is made
+  --ssl-cert       SSL Certificate used to start the server in HTTPS mode
+  --ssl-key        SSL Key used to start the server in HTTPS mode
 
 EXAMPLES
   $ zcli themes:preview ./copenhagen_theme
   $ zcli themes:preview ./copenhagen_theme --port=9999
   $ zcli themes:preview ./copenhagen_theme --no-livereload
+  $ zcli themes:preview ./copenhagen_theme --ssl-cert localhost.crt --ssl-key localhost.key
 ```
 
+### Previewing in Safari
```

Comment:
This could become an issue in other browsers as well so might be best not to single out Safari and maybe name this section `HTTPS`.